src/configure.py

```diff
@@ -567,6 +567,18 @@ def load_arguments():
                          help="Enable the interactive mode of libyt. This activates python prompt and does not shut down a simulation when there are errors in an inline python script. Must compile libyt with INTERACTIVE_MODE. Must enable <--libyt>.\n"
                        )
 
+    parser.add_argument( "--libyt_reload", type=str2bool, metavar="BOOLEAN", gamer_name="LIBYT_RELOAD",
+                         default=False,
+                         depend={"libyt":True},
+                         help="Call the reloading script feature in libyt. This activates reloading script feature, which can reload and update the script. Must compile libyt with INTERACTIVE_MODE. Must enable <--libyt>.\n"
```

Comment:
Simplify
"Call the reloading script feature in libyt. This activates reloading script feature, which can reload and update the script."
as
"Allow for reloading libyt scripts during runtime."